backend/routes/auth.py

```
@@ -73,7 +73,9 @@ def login():
 
     # Check if the user exists and the password is correct
     if user and user.check_password(data["password"]):
-        access_token = create_access_token(identity=user.id)
+        access_token = create_access_token(
+            identity=str(user.id)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)